lib/interfaces/bloc.dart

```diff
@@ -0,0 +1,4 @@
+abstract class Bloc{
+  Future<void> initialize();
+  void dispose();
+}
```

Comment:
You missed a newline at end of file here.

Reply:
Done!